support-lib/wasm/djinni_wasm.cpp

```diff
@@ -180,6 +162,10 @@ EM_JS(void, djinni_init_wasm, (), {
             Module.protobuf[name] = proto;
         };
 
+        Module.djinni_throw_cpp_exception = function(e) {
+            throw (e instanceof Error) ? e : new Error("djinni: " + e);
```

Comment:
With the below suggestion I believe you can do:
```suggestion
            throw new Error("djinni: " + e);
```

I also think that using the terms "native" or "wasm" might be easier to recognize for folks who aren't familiar with djinni.